src/plot_utils.py

```diff
@@ -633,7 +657,7 @@ def zmatrix(data_df, clustermap_kws=None, row_ordering=None,
         df = df.ix[:, col_ordering]
         df = df.ix[row_ordering, :]
         del clustermap_kws['pivot_kws']
-        fig, ax = plt.subplots()
+        _fig, ax = plt.subplots()
```

Comment:
That's right, `plt.subplots((r,c))` is factory method returning fresh figure and grid of `r,c` axes in the figure . On the other hand `plt.plot, plt.gca, plt.bar` etc alter global state and usage is discouraged.